cmd/collect_reward.go

```diff
@@ -0,0 +1,72 @@
+package cmd
+
+import (
+	"log"
+
+	"github.com/0chain/gosdk/zboxcore/sdk"
+	"github.com/spf13/cobra"
+)
+
+type Provider int
+
+const (
+	ProviderMiner Provider = iota
+	ProviderSharder
+	ProviderBlobber
+	ProviderValidator
+	ProviderAuthorizer
```

Comment:
We can use the Provider defined in sdk so that we don't have to maintain duplicate Providers everywhere it needs.